src/2023/code-of-conduct/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Code of Conduct | Helvetic Ruby Conference 2023</title>
+    <link rel="stylesheet" href="../styles/styles.css">
+  </head>
+  <body>
+    <div class="cover">
+      <header class="constrain">
+        <nav class="menu top-menu">
+          <ul class="wrap">
+            <li><a href="../">Helvetic Ruby 2023</a></li>
+            <li>Code of Conduct</li>
+          </ul>
+        </nav>
+      </header>
+      <main class="constrain region">
+        <div class="stack prose">
+          <h1>Code of Conduct</h1>
+          <p>
+            Our goal is to maintain an inclusive, safe, helpful, and friendly community for
+            everyone, regardless of experience, gender identity and expression, sexual
+            orientation, disability, personal appearance, body size, race, ethnicity, age,
+            religion, nationality, or other defining characteristic.
+          </p>
+
+          <p>
+            This code and related procedures also apply to unacceptable behavior occurring
+            outside the scope of community activities, in all community venues — online and
+            in-person — as well as in all one-on-one communications, and anywhere such
+            behavior has the potential to adversely affect the safety and well-being of
+            community members.
+          </p>
+
+          <h2>Expected Behavior</h2>
+
+          <ul>
+            <li>Be welcoming.</li>
+            <li>Be kind.</li>
+            <li>Look out for each other.</li>
+          </ul>
+
+          <h2>Unacceptable Behavior</h2>
+
+          <ul>
+            <li>Conduct or speech which is sexist, racist, homophobic, transphobic, ableist,
+              or otherwise discriminatory or offensive in nature. This may include:
+              <ul>
+                <li>Unwelcome, suggestive, derogatory, or inappropriate nicknames or terms.</li>
+                <li>Disrespectful jokes, innuendo, or dismissive language and attitude.</li>
+              </ul>
+            </li>
+            <li>Intimidation or harassment (online or in-person).</li>
+            <li>Inappropriate attention or contact. Be aware of how your actions affect
+              others. If you are making someone uncomfortable, stop.</li>
+            <li>Sustained disruptions.</li>
+            <li>Violence, threats of violence, or violent language.</li>
+            <li>Other disruptive or unsafe behavior.</li>
+          </ul>
+
+          <h2>Enforcement</h2>
+
+          <p>
+            If you are the subject of or witness to violations of this Code of Conduct,
+            please let us know in whatever way you feel comfortable: you can speak with us
+            directly or email <a href="email:conduct@helvetic-ruby.ch">conduct@helvetic-ruby.ch</a>. Organizers will take any action they deem
```

Comment:
Isn't it `href="mailto: [...]"`?

Reply:
Yes, I forgot to check when I added the link. 👍